views/backend/login.php

```diff
@@ -17,9 +17,13 @@
 
             <?php
                 if(isset($_POST['submit'])){
-                    $email = htmlspecialchars(trim($_POST['email']));
-                    $password = htmlspecialchars(trim($_POST['password']));
-
+                    if(isset($_POST['email'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)